prediction/README.md

```diff
@@ -2,7 +2,7 @@
 
 You can host your trained machine learning models on AI Platform to infer target values for new data.
 
-Currently, AI Platform can serve models generated by [scikit-learn](https://scikit-learn.org), [xgboost](https://xgboost.readthedocs.io/en/latest/),
+Currently, AI Platform can serve models generated by [scikit-learn](https://scikit-learn.org), [xgboost](https://xgboost.readthedocs.io/en/latest/), [PyTorch](https://pytorch.org/), 
```

Comment:
This isn't officially supported by AI Platform and that blog post currently doesn't work.
https://github.com/GoogleCloudPlatform/cloudml-samples/issues/410